ux.symfony.com/templates/components/DocsLink.html.twig

```diff
@@ -5,9 +5,9 @@
                rel="{{ isExternal ? 'external noopened noreferrer' }}"
             >{{ title }}</a>
             {% if icon|default %}
-                <twig:ux:icon name="{{ icon }}" class="DocsLink_arrow"/>
+                <twig:ux:icon name="{{ icon }}" class="Icon DocsLink_arrow"/>
```

Comment:
Do you need to re-mention `Icon` again even if you configured `default_icon_attributes`? 

I believe people will assume that `default_icon_attributes.class` and `class` attr will merge, no?

Reply:
They do not ... and as much i understand the need (especially here).... it's really more complicated than it seems.

Right now, i'd suggest to use the IconSets for this kind of usage... 

Here i just wanted to remove the renderer decorator and start using ux_icons config, but it's sub-optimal i agree